Contributing/readme.md

```diff
@@ -26,4 +26,34 @@ import SectionHeader from '../../../../utilities/SectionHeader/SectionHeader';
 
 When adding a SVG to the codebase, be sure to it SVGs through this optimizer: https://jakearchibald.github.io/svgomg.
 
-It will help strip out a bunch of unnecessary junk that Adobe Illustrator, Sketch (insert your favorite vector program), etc add to the code. The default settings are usually just fine and you’ll usually see around like a 50% drop in file size for most SVGs!
\ No newline at end of file
+It will help strip out a bunch of unnecessary junk that Adobe Illustrator, Sketch (insert your favorite vector program), etc add to the code. The default settings are usually just fine and you’ll usually see around like a 50% drop in file size for most SVGs!
+
+## Tests
+
+### Data attributes
+
+When writing tests, it's handy to check for the existence of specific HTML elements, but we want to avoid adding an id or class to an element if it's only used specifically for testing. For those instances, we'll use `data-*` attributes.
+
```

Comment:
I wonder if instead of `data-*` we just declare it should always be `data-test` and the value it equals is what's specific to the element, like in the examples below. This could avoid having funky named data attributes where it may not be obvious they're specifically for testing? 

Thoughts?

Not sure if there's a scenario for needing a hook for testing that wouldn't be good to name it `data-test` 🤷‍♂️

Reply:
Yeah, that's a fair point. And I like being able to do a simple [GitHub search](https://github.com/DoSomething/phoenix-next/search?q=data-test&unscoped_q=data-test) to find all of these, which we can't do if it's a prefix for multiple possible attributes. (From a quick grep, it already looks like we've got a mix of `data-test`, `data-ref`, `data-contentful-id`, and `data-id`).

I could see `data-test` (and, optionally, `data-id`) covering all of our needs. For example, we could swap [`data-contentful-id`](https://github.com/DoSomething/phoenix-next/blob/dc4332d497c59477268165cf0956c59302d50598/resources/assets/components/utilities/ContentfulEntryLoader/ContentfulEntryLoader.js#L168) here for `data-test="ContentfulEntry" data-id={id}`...

Reply:
I think explicitly encouraging to use `data-test` is a great idea. Does this hook have any other purpose besides testing?

https://github.com/DoSomething/phoenix-next/blob/dc4332d497c59477268165cf0956c59302d50598/cypress/integration/campaign-post.js#L62

Reply:
Nope! Just used for Cypress & Ghost Inspector tests.

Reply:
Ah, your comment hadn't loaded on the page when I asked my question you totes answered in the comment above. I'll add the prompt to use `data-test` and mention how `data-contentful-id` should be refactored to keep consistent. I'd encourage us to get into the habit now to have less tests to manually change?